src/index.html

```diff
@@ -1,13 +1,61 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="page-html">
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width,
+        user-scalable=no,
+        initial-scale=1.0,
+        maximum-scale=1.0,
+        minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+  </head>
+
+  <body class="reset">
+    <div class="product-card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="image-imac"
+        class="product-card__img"
+      >
+
+      <h2 class="product-card__name reset">
```

Comment:
Why do you add `reset` class to every element? Let's delete this
```suggestion
      <h2 class="product-card__name">
```